docs/one-click-integrations/github-integration.md

```diff
@@ -74,14 +82,31 @@ The following is a detailed description of how the Pulse GitHub integration auto
 
 -   Pulse associates all GitHub teams of the person who creates a Git tag with the corresponding deployment, excluding teams with less than two members. Pulse only takes changes to GitHub teams into account on Git tags created after those changes.
 
-### Use the CLI or API
+### Use the CLI or API (don't detect deployments automatically)
 
 -   Pulse doesn't detect deployments automatically using GitHub events.
 
     This is useful if none of the automatic deployment detection strategies match your workflow and you must have control over the way Pulse tracks your deployments.
 
 -   In this case, you must send to Pulse the information about your **deployments** and the corresponding **changes** using the [Pulse CLI](../cli/cli.md) or the [Ingestion API](https://ingestion.pulse.codacy.com/v1/api-docs).
 
+## Automatic incident detection strategies {: id="incident-detection-strategy"}
+
+The following is a detailed description of how the Pulse GitHub integration automatically detects incidents:
+
+### Use pull request reverts (based on default branch)
+
+-   Pulse bases incident detection on [pull request reverts](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/reverting-a-pull-request).
+-   Pulse considers an incident any pull request that **targets the default branch** of the repository merged from a branch which name starts with `revert-pr-`, getting the number of the reverted pull request from the branch name, `revert-pr-<pull request number>`. If you change the name of the branch created by GitHub when you revert a pull request, Pulse may not be able to obtain the incident data correctly.
```

Comment:
In GitHub it's not possible to customize the name of a branch that was created via the revert feature. But I guess the warning still makes sense, since users can manually follow this convention.

Reply:
Thanks! I'll keep the warning. 👍